docs/rules/template-tag-spacing.md

```diff
@@ -0,0 +1,78 @@
+# Enforce Usage of Spacing in Tagged Template Literals (template-tag-spacing)
```

Comment:
Can you update this to match [this description](https://github.com/eslint/eslint/pull/7913/files#diff-8dbaf8148ca54cc6af73a2dd5accc3b0R15)?

i.e. `Require or disallow spacing between template tags and their literals (template-tag-spacing)`